README.md

```diff
@@ -63,1498 +104,468 @@ Delegating a capability is like the statement `message`. Task is akin to `messag
+The REQUIRED Arguments (`args`) field, MAY contain any parameters expected by the Command. The Subject MUST be considered the authority on the shape of this data. This field MUST be representable as a map or keyword list.
 
-  # All the other metadata
-  meta    {String: any}
+The Arguments MUST pass validation of the Policies on all of the [UCAN Delegations][UCAN Delegation] in the [Proofs] field. If any Policy reports failure, the Invocation MUST be rejected.
 
-  # Principal that issued this receipt.
-  # If omitted issuer is inferred from 
-  # the invocation task audience.
-  iss     optional Principal
+#### Nonce
+[Nonce]: #nonce
 
-  # When issuer is different from executor
-  # this MUST hold a UCAN delegation chain 
-  # from executor to the issuer.
-  # This field SHOULD be omitted executor is an issuer.
-  prf     [&UCAN]
+The REQUIRED `nonce` field MUST include a random nonce. This field ensures that multiple (non-idempotent) invocations are unique. The nonce SHOULD be empty (`0x`) for Commands that are idempotent (such as deterministic Wasm modules or standards-abiding HTTP PUT requests).
 
-  # Signature from the `iss`.
-  s       Varsig
-}
-```
+### Proofs
+[Proofs]: #proofs
 
-## 8.2 Fields
+The `prf` field defines all [Delegation]s required to prove that this Invocation has an unbroken authorization chain.
 
-### 8.2.1 Ran Invocation
+### Provenance
+[Provenance]: #provenance
 
-The `ran` field MUST include a link to the [Invocation] that the Receipt is for.
+#### Cause
+[Cause]: #cause
 
-### 8.2.2 Output
+The OPTIONAL `cause` field is a provenance claim describing which [Receipt] requested it. This is helpful for tracking chains of Invocations.
 
-The `out` field MUST contain the value output of the invocation in [Result] format.
+#### Expiration
+[Expiration]: #expiration
 
-### 8.2.3 Effect
+The OPTIONAL field `exp` defines when the Invocation SHOULD time out. This is both expressive (defines a timeout, which is a best practice), and prevents replays.
 
-The OPTIONAL `fx` field, if present MUST be set to the caused [Effect]. The [Executor] SHOULD invoke contained [Task] to progress a workflow execution.
+#### Issued At
+[Issued At]: #issued-at
 
-If `fx` does not contain OPTIONAL `join` field, it denotes completion of the current execution thread.
+The OPTIONAL `iat` field MAY contain an issuance timestamp. This time SHOULD NOT be trusted; it is only a claim by the Invoker of their system time. System clocks often have clock skew, or a Byzantine Invoker could claim an arbitrary time.
 
-### 8.2.4 Metadata Fields
+#### Metadata
+[Metadata]: #metadata
 
-The OPTIONAL metadata field MAY be omitted or used to contain additional data about the receipt. This field MAY be used for tags, commentary, trace information, and so on.
+The OPTIONAL `meta` field MAY include arbitrary metadata or extensible fields. For example, Wasm fuel, an internal job ID, references to GitHub Issues, and so on. This data MAY be used by the Executor.
```

Comment:
this "MAY be used" is a little confusing relative to this passage in the [high level spec](https://github.com/ucan-wg/spec/blob/9b50374ba600eea56a697f204f870cd36fba37c7/README.md#metadata): 

>  The data contained in this map MUST NOT be semantically meaningful to delegation chains.

Reply:
This isn't for delegation chains